utils/cargo-gbuild/src/cli.rs

```diff
@@ -35,22 +35,63 @@ pub struct GBuild {
     /// Directory for all generated artifacts
     #[clap(short, long)]
     pub target_dir: Option<PathBuf>,
+
+    /// If disable wasm-opt
+    pub no_wasm_opt: bool,
+
+    /// If enable meta build
+    #[clap(short, long)]
+    pub meta: bool,
 }
 
 impl GBuild {
     /// Build program
     pub fn build(&self) -> Result<()> {
-        let _target = self
-            .target_dir
-            .clone()
-            .unwrap_or(manifest::parse_target(&self.manifest_path)?);
-
-        // TODO: inheirt the logging format of cargo.
-        //
-        // for example:
-        //
-        // Compiling program-name v0.0.1 ( /path/to/the/program )
-        tracing::info!("Program artifacts have been output in {_target:?} !");
+        self.cargo()?;
+        self.collect()
+    }
+
+    /// Process the cargo command.
+    ///
+    /// NOTE: only supports release build.
```

Comment:
better `TODO: should support debug builds.` :)

Reply:
https://github.com/gear-tech/gear/pull/3835/commits/fa2c62b25105fb169eab37713f81121569177d90